backend/module/eCampCore/config/module.config.php

```diff
@@ -60,13 +60,13 @@
         // Use lazy services (service proxies) for expensive constructors or in case circular dependencies are needed
```

Comment:
```suggestion
        // Use lazy services (service proxies) for expensive constructors or in case circular dependencies are needed
        // #572 please remember if we want to use lazy services we need to remove the final methods from AbstractEntityService
```